### PR TITLE
Target pin

### LINKED ISSUE #90: Target A Specific Pin
As a user I want to be able to choose a specific pin as my target

### PR DESCRIPTION
**Pull request by jake-shasteen**
Resolves #90 

Add reducer, action, constants for setting pin
Add targetPin to store
Add options in mapview and listview for setting target pin
Render targeted pin differently in map, ar, and list views

### CODE REVIEW

File: app/components/AR.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ const HTML = `
 
 const BRIDGE_INJECT_SCRIPT = `
   var targetLocIdx = 0;
```
Comment (bignell) [inline app/components/AR.js]: `targetLocIndex` might be clearer, but that's just me. And/or do we want to standardize on `pins` over `locs`?

**Comment by bignell** [comment]
@jake-shasteen Reviewed and tested—works great.

File: app/components/PinListItem.js [hunk 2]
```diff
@@ -119,4 +127,7 @@ const style = StyleSheet.create({
     color: 'black',
     alignSelf: 'center',
   },
+  target: {
+    backgroundColor: 'red',
```
Comment (asu91) [inline app/components/PinListItem.js]: can we not have the targeted pin as red?

File: app/components/AR.js (hunk 1 above)
Comment (jake-shasteen) [inline app/components/AR.js]: They're only called locs in the context of the webview that's overlayed on top of the camera, as a way to keep the labels separate across the webview bridge.

File: app/components/PinListItem.js (hunk 2 above)
Comment (jake-shasteen) [inline app/components/PinListItem.js]: We can change it when we go through and do polish and styling. Solidify the color scheme and whatnot. I wanted to demonstrate how one would differentiate the target pin in both the map and list contexts, but the actual color or design will have to be done by whoever handles that issue.

File: app/components/AR.js (hunk 1 above)
Comment (bignell) [inline app/components/AR.js]: Seems reasonable enough. :+1:

**Comment by asu91** [comment]
greatness

**Pull request merged**